openedx/features/enterprise_support/api.py

```diff
@@ -268,7 +293,7 @@ def consent_needed_for_course(user, course_id):
     return consent_necessary_for_course(user, course_id)
 
 
-def get_enterprise_consent_url(request, course_id, user=None, return_to=None):
+def get_enterprise_consent_url(request, course_id, user=None, return_to=None, return_to_url=None):
```

Comment:
@haikuginger Is there any way to use `return_to` for the new use case instead of adding a new parameter? If not, we'll need to update the list of arguments in the docstring below to include `return_to_url`.